packages/opentelemetry-plugin-grpc-js/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@opentelemetry/plugin-grpc-js",
+  "private": true,
```

Comment:
why true ?

Reply:
This prevents it from being accidentally published before it is ready